Dockerfile

```diff
@@ -0,0 +1,57 @@
+ARG PY_VERSION=3.9
+
+FROM amazon/aws-lambda-python:$PY_VERSION as install-stage
+
+# Declare it a second time so it's brought into this scope.
+ARG PY_VERSION=3.9
+
+# Install the Python packages necessary to install the Lambda dependencies.
+RUN python3 -m pip install --no-cache-dir \
+    pip \
+    setuptools \
+    wheel \
+  # This version of pipenv is the minimum version to allow passing arguments
+  # to pip with the --extra-pip-args option.
+  && python3 -m pip install --no-cache-dir "pipenv>=2022.9.8"
+
+WORKDIR /tmp
+
+# Copy in the dependency files.
+COPY src/py$PY_VERSION/ .
+
+# Install the Lambda dependencies.
+#
+# The --extra-pip-args option is used to pass necessary arguments to the
+# underlying pip calls.
+RUN pipenv sync --system --extra-pip-args="--no-cache-dir --target ${LAMBDA_TASK_ROOT}"
```

Comment:
In the commit message for [the commit that added `--system`](https://github.com/cisagov/skeleton-aws-lambda-python/pull/3/commits/a62a30076034012487a59518c76bc9371a28ed75) you mentioned that using the system python may cause problems (if I understand correctly what you wrote).  Should that caution be added to the comment here?

Reply:
Re-reading it I mention that using the virtual environment automatically generated by Pipenv by default is the problem. Using `--system` is what prevents that default behavior so that we get desired behavior. Does that make sense? I can re-word the commit message to improve clarity if needed.

Reply:
OK, rereading the commit message now I get it.